examples/hello/lib/functions.dart

```diff
@@ -16,4 +16,10 @@ import 'package:functions_framework/functions_framework.dart';
 import 'package:shelf/shelf.dart';
 
 @CloudFunction()
-Response function(Request request) => Response.ok('Hello, World!');
+Response handleGet(Request request) => Response.ok('Hello, World!');
```

Comment:
Confused by this change...

Reply:
oops! was undoing a comment that I had added. Undid too much, I guess -- thanks for catching!